inc/sf_c_util/servicefabric_enums_to_strings.h

```diff
@@ -1,5 +1,4 @@
 // Copyright (C) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
>ed under t [](http://example.com/codeflow?start=9&length=10)

Seems like the file has been generated before the Licensed portion was added to the .cs

Reply:
Or the header generation code does not have it. One of the 2!

Reply:
I am equally puzzled. It is really unexplainable, but I fexd it.